src/converter/entities.ts

```diff
@@ -69,6 +68,29 @@ export const convertComputer = (
   });
 };
 
+interface NetworkAddress {
+  mac?: string;
+  ip?: string;
+}
+
+export function convertNetworkAddressesToArray(
+  networkAddressData: NetworkAddress[] | undefined | null,
+  key: string,
+): string[] {
+  // undefined or null data is returned as undefined
+  if (networkAddressData === undefined || networkAddressData === null) {
```

Comment:
```suggestion
  if (!Array.isArray(networkAddressData)) {
```

The above suggestion would simplify the rest of the function.